bin/MadGraph5_aMCatNLO/cards/production/13TeV/ggh01_M125_Toa01a01_Tomumutautau/ggh01_M125_Toa01a01_M15_Tomumutautau/ggh01_M125_Toa01a01_M15_Tomumutautau_run_card.dat

```diff
@@ -0,0 +1,290 @@
+  -1.0	=  htjmax  ! maximum jet HT=Sum(jet pt)
+  0.0	=  ihtmin   !inclusive Ht for all partons (including b)
+  -1.0	=  ihtmax   !inclusive Ht for all partons (including b)
+  0.0	=  ht2min  ! minimum Ht for the two leading jets
+  0.0	=  ht3min  ! minimum Ht for the three leading jets
+  0.0	=  ht4min  ! minimum Ht for the four leading jets
+  -1.0	=  ht2max  ! maximum Ht for the two leading jets
+  -1.0	=  ht3max  ! maximum Ht for the three leading jets
+  -1.0	=  ht4max  ! maximum Ht for the four leading jets
+#***********************************************************************
+# Photon-isolation cuts, according to hep-ph/9801442                   *
+# When ptgmin=0, all the other parameters are ignored                  *
+# When ptgmin>0, pta and draj are not going to be used                 *
+#***********************************************************************
+  0.0	=  ptgmin  ! Min photon transverse momentum
+  0.4	=  R0gamma  ! Radius of isolation code
+  1.0	=  xn  ! n parameter of eq.(3.4) in hep-ph/9801442
+  1.0	=  epsgamma  ! epsilon_gamma parameter of eq.(3.4) in hep-ph/9801442
+  True	=  isoEM  ! isolate photons from EM energy (photons and leptons)
+#*********************************************************************
+# WBF cuts                                                           *
+#*********************************************************************
+  0.0	=  xetamin  ! minimum rapidity for two jets in the WBF case  
+  0.0	=  deltaeta  ! minimum rapidity for two jets in the WBF case 
+#*********************************************************************
+# KT DURHAM CUT                                                      *
+#*********************************************************************
+  -1.0	=   ktdurham        
+   0.4	=   dparameter 
+ #*********************************************************************
+# maximal pdg code for quark to be considered as a light jet         *
+# (otherwise b cuts are applied)                                     *
+#*********************************************************************
+  4	=  maxjetflavor     ! Maximum jet pdg code
+#*********************************************************************
+# Jet measure cuts                                                   *
+#*********************************************************************
+  0.0	=  xqcut    ! minimum kt jet measure between partons
+#*********************************************************************
+#
+#*********************************************************************
+# Store info for systematics studies                                 *
+# WARNING: If use_syst is T, matched Pythia output is                *
+#          meaningful ONLY if plotted taking matchscale              *
+#          reweighting into account!                                 *
+#*********************************************************************
+  True	=  use_syst       ! Enable systematics studies
+#
```

Comment:
Hello please remove the lines from below. It sometimes creates some parsing issues during production